_rest_routes.inc.php

```diff
@@ -5314,9 +5314,82 @@
      *      security={{"openemr_auth":{}}}
      *  )
      */
-    "GET /api/appointment/:eid" => function ($eid) {
+    "GET /api/appointment/:euuid" => function ($euuid) {
```

Comment:
If is possible add typo for the funciton argument:
function (string $puuid) { ... }